### PR TITLE
Fix wrong switch name

### LINKED ISSUE #16336: System.Xml.IgnoreEmptyKeySequences without Switch at the beggining
In the table, all switches are written with Switch in the begging except System.Xml.IgnoreEmptyKeySequences. But in fact, we must write Switch.System.Xml.IgnoreEmptyKeySequences to make it work. Please fix this issue, we spend two days understanding why this does not work.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e59350ee-7837-34f8-b50e-c9c136720155
* Version Independent ID: 19747ce7-da26-bf2e-5066-1df11e61b0d0
* Content: [&lt;AppContextSwitchOverrides&gt; Element](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element#feedback)
* Content Source: [docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md](https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md)
* Product: **dotnet-framework**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #16336

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
 |`Switch.UseLegacyToolTipDisplay`|Controls whether tooltips are displayed when a user hovers the mouse cursor over a WPF control (`true`), or whether they are displayed both on keyboard focus and via keyboard shortcut key (`false`, the default behavior). For applications running on .NET Framework 4.8 but targeting previous versions of the .NET Framework, enabling both keyboard focus and shortcut key support requires that `Switch.UseLegacyAccessibilityFeatures`, `Switch.UseLegacyAccessibilityFeatures.2`, and `Switch.UseLegacyAccessibilityFeatures.3` all be set to `false`.|.NET Framework 4.8|
-|`System.Xml.`<br /><br /> `IgnoreEmptyKeySequences`|Controls whether empty key sequences in compound keys are ignored by XSD schema validation. For more information, see [Mitigation: XML Schema Validation](../../../migration-guide/mitigation-xml-schema-validation.md).|.NET Framework 4.6|  
+|`Switch.System.Xml.`<br /><br /> `IgnoreEmptyKeySequences`|Controls whether empty key sequences in compound keys are ignored by XSD schema validation. For more information, see [Mitigation: XML Schema Validation](../../../migration-guide/mitigation-xml-schema-validation.md).|.NET Framework 4.6|
```
Comment (gewarren) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]:
```suggestion
|`Switch.System.Xml`<br /><br /> `IgnoreEmptyKeySequences`|Controls whether empty key sequences in compound keys are ignored by XSD schema validation. For more information, see [Mitigation: XML Schema Validation](../../../migration-guide/mitigation-xml-schema-validation.md).|.NET Framework 4.6|  
```
?

**Review by gewarren — APPROVED** [review]
But see question about period.

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: The period is needed

Comment (mairaw) [reply docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: But your comment made me realize that this was poorly formatted.
I've removed the extra line

image

**Pull request merged**